.travis.yml

```diff
@@ -2,3 +2,7 @@ language: java
 jdk:
   - oraclejdk8
 sudo: required
+env:
+  - HUDI_QUIETER_LOGGING=1
+services:
```

Comment:
is this related to our docker/integ test stuff ? i.e can this be merged before that PR?